pkg/tfbridge/transforms.go

```diff
@@ -25,6 +24,11 @@ import (
 // TransformJSONDocument permits either a string, which is presumed to represent an already-stringified JSON document,
 // or a map, which will be transformed into its JSON representation.
 func TransformJSONDocument(v resource.PropertyValue) (resource.PropertyValue, error) {
+	// We can't marshal properties that contain unknowns. Turn these into an unknown value instead.
+	if v.ContainsUnknowns() {
+		return resource.MakeComputed(resource.NewStringProperty("")), nil
```

Comment:
I'm not sure I follow the implications of returning `""` here - is that the same as what the value would have looked like prior to us adding this `TransformJSONDocument`?

Reply:
We're not returning `""`; we're returning `computed<>`. The `""` is just a formality, and will never be displayed.